README.md

```diff
@@ -69,7 +74,7 @@ Other options that are supplied are passed query options if they are known, othe
 
 - Only `SELECT` statements are supported.
 - Statements using parameters must use the `:param` syntax and must have a corresponding entry in the `params` map.
-- The only standard 
+- The only standard
```

Comment:
It is incomplete, but I don't remember what I was typing, so feel free to remove it.